Cargo.toml

```diff
@@ -62,6 +67,12 @@ tiny-skia = { version ="0.6.*", optional = true }
 strum = { version ="0.24.*", features = ["derive"], optional = true }
 dot-writer = { version = "0.1.*", optional = true }
 
+ion-rs = {git="https://github.com/amazon-ion/ion-rust"}
```

Comment:
Is it pointing to the GitHub repo because there's a feature not yet released by `ion-rs`?

Reply:
Yes. In some cases (several of my test cases), the ion reader would fail. All that I've seen are fixed by https://github.com/amazon-ion/ion-rust/pull/552